app/src/main/java/com/hyperether/getgoing/activity/ShowLocationActivity.java

```diff
@@ -908,4 +926,32 @@ public void openGPSSettings() {
         Intent i = new Intent(ACTION_LOCATION_SOURCE_SETTINGS);
         startActivityForResult(i, Constants.REQUEST_GPS_SETTINGS);
     }
+
+    private void backgroundTrackNotification(int notificationID){
+        NotificationCompat.Builder builder = new NotificationCompat.Builder(this);
+        builder.setSmallIcon(R.drawable.ic_launher);
+        builder.setContentTitle("GetGoing notification");
+        builder.setContentText("Location tracking in background");
+        builder.setAutoCancel(true);
+        builder.setDefaults(Notification.DEFAULT_VIBRATE);
+
+        PendingIntent pendingIntent = PendingIntent.getActivity(this, 0, this.getIntent(), PendingIntent.FLAG_UPDATE_CURRENT);
+
+        builder.setContentIntent(pendingIntent);
+
+
+        notificationManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        notificationManager.notify(notificationID,builder.build());
+        notificationExist = true;
+
```

Comment:
@NikolaHadzicNs remove those empty lines

Reply:
I removed empty lines and vibration and defined text in xml.